airgun/views/cloud_insights.py

```diff
@@ -117,6 +119,23 @@ def is_all_selected(self):
         return self.checkbox.selected
 
 
+class MenuToggleSelect(PF5Select):
+    """
+    This class is PF5 implementation of the Select component within the new PF5 structure
+    Which is MenuToggle->Select and not just Select as it was in PF4.
+    """
+
+    BUTTON_LOCATOR = './/button[contains(@class, "pf-v5-c-menu-toggle")]'
+    DEFAULT_LOCATOR = (
+        './/div[contains(@class, "pf-v5-c-menu") and @data-ouia-component-type="PF5/Select"]'
+    )
+    ROOT = ParametrizedLocator('{@locator}/..')
+    ITEMS_LOCATOR = ".//ul[contains(@class, 'pf-v5-c-menu__list')]/li"
+    ITEM_LOCATOR = (
+        "//*[contains(@class, 'pf-v5-c-menu__item') and .//*[contains(normalize-space(.), {})]]"
+    )
```

Comment:
@ColeHiggins2 could you please use it like this. We can reuse most of the MenuToggleSelect body.
```suggestion
from airgun.views.all_hosts import MenuToggleSelect

class MenuToggleSelectParamLocator(MenuToggleSelect):
    """
    something about why we use parametrized locator here
    """
    ROOT = ParametrizedLocator('{@locator}/..')
```

Reply:
reverting back to original implementation because of circular imports